src/script/components/MessagesList/Message/ContentMessage/asset/VideoAsset.tsx

```diff
@@ -153,6 +205,7 @@ const VideoAsset: React.FC<VideoAssetProps> = ({
               poster={videoPreview?.url}
               onError={event => {
                 setVideoPlaybackError(true);
+                amplify.publish(WebAppEvents.ANALYTICS.EVENT, EventName.MESSAGES.VIDEO.PLAY_FAILED);
```

Comment:
There are 4 place where we logged `PLAY_FAILED` event.
Just curious if it will only log once per error.

Reply:
I've checked and it shouldn't fire a few at once. 

1. fires when mime type is not supported (the simple check)
2. fires when a video is not playable (the advanced check)
3. fires when something fails in the `try{}` block inside `onPlayButtonClicked`  (e.g. `getAssetUrl` function)
4. fires on the video error callback - I guess now it's not gonna be a common case (because we handled unplayable videos before that), but in case something fails, then we can log it 

I wanted to add logs in many places, so that we can later analyse how many videos were played successfully, how many ended with "unsupported mime type", and how many failed due to unsupported codecs.

Reply:
Thanks for explanation 🚀